src/Umbraco.Web/Features/EnabledFeatures.cs

```diff
@@ -0,0 +1,22 @@
+namespace Umbraco.Web.Features
+{
+    /// <summary>
+    /// Represents enabled features.
+    /// </summary>
+    internal class EnabledFeatures
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EnabledFeatures"/> class.
+        /// </summary>
+        public EnabledFeatures()
```

Comment:
empty ctor is not needed?

Reply:
where else would I put my xml comments??! 😄 
... but yes.. we could prolly remove that :)